tests/acceptance/RobotLab/__init__.robot

```diff
@@ -3,5 +3,6 @@ Suite Setup       Start New RobotLab Server
 Suite Teardown    Terminate All Jupyter Servers
 Test Teardown     Reset JupyterLab and Close
 Force Tags        ui:lab
+Library           SeleniumLibrary
```

Comment:
I forgot to remove this line. No longer required. The error that was supposed to be fixed by "Close all browsers" was actually the issue where JupyterLab started with previous Untitled.ipynb instead of the launcher.

Reply:
Gotcha. It's probably fine :)